__tests__/app.test.js

```diff
@@ -325,13 +336,49 @@ describe("/api/users", () => {
           });
         });
     });
+  });
+});
 
-    test("GET 404: should return status code 404 for invalid path for endpoint", () => {
+describe("/api/articles(topic)", () => {
+  describe("GET", () => {
+    test("GET 200: should return articles filtered by topic if topic query is provided", () => {
+      const topic = "mitch";
       return request(app)
-        .get("/api/nonsense")
+        .get(`/api/articles?topic=${topic}`)
+        .expect(200)
+        .then((response) => {
+          expect(response.body).toBeInstanceOf(Array);
```

Comment:
No need to check for an array as we are using a forEach after this proves it is an array.
It's also essential that we have a length check here, otherwise if the array was empty your expect wouldn't run and the test would auto pass.